### ISSUE TITLE
Haveged required on docker host

### ISSUE THREAD

**Issue by Duvel** [passbolt/passbolt_docker#121]
I was trying to start passbolt on docker and it failed like in the issue #117. To get it working I installed haveged on the docker host. I didn't expect anything to be required on the host, but apparently it is.

### What you expected to happen

1. No dependency on the host
2. At least a hint, why it wasn't starting
3. documentation mentioning it should be installed on the host

**Comment by dlen** [comment]
Hi @Duvel, 
Indeed haveged/rng-tools might be required if your entropy pool filling is too slow.

> * No dependency on the host

This is a hard one and it depends completely on your platform. Passbolt relies on GPG and GPG requires entropy to perform crypto operations. If your system provides enough entropy let's say using a hardware number generator then you will be fine. However in most of the cases you won't have such hardware available so haveged/rng-tools are a fair trade-off for most virtual environments as it is easy to setup. 
As a rule of thumb you can consider that when dealing with crypto stuff on virtual environments you will hit the entropy wall at some point.
Bonus: If the limitation is just installing dependencies on the host you could run haveged/rng-tools on a separated privileged docker container to fill the entropy pool.

> * At least a hint, why it wasn't starting

It could be possible to print a message before the serverkey generation step if the entropy pool is low. I agree this could be done and your feedback will be considered!

> * documentation mentioning it should be installed on the host

It is already mentioned on this repo [README](https://github.com/passbolt/passbolt_docker#requirements) and on [passbolt help](https://help.passbolt.com/hosting/install/ce/docker) docker section.

Hope this clears a bit the situation.

**Comment by Duvel** [comment]
Hi @dlen 

Thx for the quick response.

> It is already mentioned on this repo [README](https://github.com/passbolt/passbolt_docker#requirements) and on [passbolt help](https://help.passbolt.com/hosting/install/ce/docker) docker section.

The documentation here is talking about optional and speeding up and not a requirement. Probably the reason why I skipped it the previous time I installed passbolt and forgot about it :(.

I'm trying to run passbolt in a docker cluster based on nodes on virtual machines, so that could be why I don't have enough entropy. I will look into adding enough entropy on all nodes, it could run on. Working with enough entropy is not something I do daily.

If it is not something that can be solved in the docker image, the hint while starting will become more important.
